app/views/articles/show.html.haml

```diff
@@ -9,6 +9,12 @@
     = @article.impressions_count
     |
     = @article.read_time
+    | Uploaded: 
+    = @article.updated_at.month
+    \-
+    = @article.updated_at.day
+    \-
+    = @article.updated_at.year
```

Comment:
please use a date formater: http://stackoverflow.com/questions/11090451/format-the-date-using-ruby-on-rails